app/scripts/lib/profile-errors.js

```diff
@@ -45,6 +45,10 @@ define(function (require, exports, module) {
     UNEXPECTED_ERROR: {
       errno: 999,
       message: t('Unexpected error')
+    },
+    NO_DISPLAYED_PROFILEIMAGE_FOUND: {
+      errno: 1049,
```

Comment:
Also, I imagine in the end this won't be the error used. Instead it'll be something like `NO_PROFILE_IMAGE_ID`